docs/permission/plugin-authors/03-adding-a-resource-permission-check.md

```diff
@@ -170,8 +174,7 @@ Now, let's create the new endpoint by editing `plugins/todo-list-backend/src/ser
 + import { createPermissionIntegrationRouter } from '@backstage/plugin-permission-node';
 - import { add, getAll, update } from './todos';
 + import { add, getAll, getTodo, update } from './todos';
-- import { todosListCreate, todosListUpdate } from './permissions';
-+ import { todosListCreate, todosListUpdate, TODO_LIST_RESOURCE_TYPE } from './permissions';
++ import { TODO_LIST_RESOURCE_TYPE } from './permissions';
```

Comment:
I actually don't think this ever worked - the `TODO_LIST_RESOURCE_TYPE` is in the frontend plugin, and this is the backend plugin. We can just use the string like we do above in order to avoid dependency issues. In reality this would be stored in a todo-list-common plugin but that balloons the complexity of the tutorial. Thoughts?

Reply:
It's tricky here because using the import would also be a best practice, so we're basically just choosing the lesser evil. I do think using the string is the lesser evil though.

Reply:
I think we do need a common package because the permission objects will need to be shared between frontend and backend for the full tutorial

Reply:
I've given this some more thought, and I'm going to bite the bullet and include a todo-list-common in the tutorial. If we scaffold the plugin and have them install it during setup, it shouldn't add too much additional load to the tutorial. I'll track it in a follow up issue.